modules/integration_aws-fsx/conf/01-used-space.yaml

```
@@ -0,0 +1,18 @@
+module: "AWS FSx"
+name: "Free Space"
+filtering: "filter('namespace', 'AWS/FSx')"
+value_unit: "Gibibyte"
+transformation: ".max(over='15m')"
+signals:
+  used_space:
+    metric: "FreeStorageCapacity"
+    filter: "filter('stat', 'mean')"
+  signal:
+    formula:
+      used_space.scale(1/1024**3)
```

How about renaming this signal and the file to match the metric name:
```suggestion
  free_space:
    metric: "FreeStorageCapacity"
    filter: "filter('stat', 'mean')"
  signal:
    formula:
      free_space.scale(1/1024**3)
```

---

done